cloudnet/src/main/java/de/dytanic/cloudnet/command/commands/CommandCreate.java

```diff
@@ -215,6 +218,10 @@ public void execute(ICommandSender sender, String command, String[] args, String
         } else
             deploy.addAll(deployments);
 
+        if (properties.containsKey("deletedFilesAfterStop")) {
+            deletedFilesAfterStop = Arrays.asList(properties.get("deletedFilesAfterStop").split(";"));
```

Comment:
Please do it like it's done with the parameters "node" and "groups", in an one-liner.